package/src/components/MessageInput/components/AudioRecorder/AudioRecordingButton.tsx

```diff
@@ -116,51 +120,6 @@ const AudioRecordingButtonWithContext = (props: AudioRecordingButtonPropsWithCon
   );
 };
 
-const areEqual = (
```

Comment:
This change is because of React. The memo is causing yellow boxes on the profiler here on the text change. But without the memo, there are green boxes. I am trying to figure out what the cause is, but this is definitely noticeable.